subscription/add-seats.md

```diff
@@ -28,7 +28,7 @@ To add seats to a monthly subscription:
 
 5. The **Billing** tab displays the number of seats currently in use.
 
-6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../orgs.md#add-a-member-to-a-team).
+6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../orgs/#add-a-member-to-a-team).
```

Comment:
I think it was broken because its in the "docker-hub" directory (not because of the .md suffix)

```suggestion
6. Navigate to the **Members** tab to add new members. For more information, see [Add a member to a team](../docker-hub/org.md#add-a-member-to-a-team).
```